app/models/fact_names/foreman_salt/fact_name.rb

```diff
@@ -0,0 +1,14 @@
+module ForemanSalt
+  # Define the class that fact names that come from Salt should have
+  # It allows us to filter facts by origin, and also to display the origin
+  # in the fact values table (/fact_values)
+  class FactName < ::FactName
+    def origin
+      'Salt'
+    end
+
+    def icon_path
+      'icons16x16/stub/black-s.png'
```

Comment:
We can use this one for now. But, foreman_salt used the salt original icon since years. I recommend to contact saltproject@vmware.com to ask for permission to use their logo.